agent-config/src/linuxX64Test/kotlin/com/epam/drill/agent/configuration/provider/EnvironmentVariablesProviderTest.kt

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2020 - 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.drill.agent.configuration.provider
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class EnvironmentVariablesProviderTest {
+
+    @Test
+    fun `to parameter name upper case`() {

```

Comment:
`to parameter name upper case` - sounds like we convert something to parameter name in upper case

I suggest to change names for clarity:
`from DRILL_UPPER_CASE_PARAM`
`from DRILL_lower_case_param`
`from DRILL_mixed_Case_pAram`
etc

+add separate case for `DRILL_` prefix  - is it required and is it _have to be_ uppercased?

Reply:
TODO - add tests - `DRILL_` is a requirement